src/index.js

```diff
@@ -0,0 +1,150 @@
+
+const getRealTimeTemp = () => {
+  //   // input: cityName -> locationQ -> lat, lon
+  //   // -> open weatherApp -> realTemp for the city (in lat, lon)
+  //   // change the element of html page: currentTemp
+  // const axios = require('axios');
+  let latitude, longitude;
+  const cityName = document.getElementById("cityName").value;
+  axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: cityName,
+      },
+    })
+    .then((response) => {
+      latitude = response.data[0].lat;
+      longitude = response.data[0].lon;
+      console.log('success in getRealTimeTemp!', latitude, longitude);
+      temperature = getTempByCity(latitude, longitude);
+      return temperature;
+
+    })
+    .catch((error) => {
+      console.log('error in getRealTimeTemp!');
+    });
+};
+
+const getTempByCity = (latitude, longitude) => {
+  axios
+    .get('http://127.0.0.1:5000/weather', {
+      params: {
+        lat: latitude,
+        lon: longitude,
+      },
+    })
+    .then((response) => {
+      console.log('success in findWeather!', response.data.main.temp);
+      const realTemp = response.data.main.temp
+      const fahrenheit = Math.round(1.8*(realTemp - 273.15) + 32)
+      console.log('here is temp in fahrenheit',fahrenheit);
+      state.currentTemp = fahrenheit
+      const tempContainer = document.querySelector('#currentTemp');
+      tempContainer.textContent = `${state.currentTemp}`;
+      changeTempColor(fahrenheit)
+      
+    })
+    .catch((error) => {
+      console.log('error in findWeather!');
+    });
+};
+
+const updateTemp = () => {
+  const tempContainer = document.querySelector('#currentTemp');
+  tempContainer.textContent = `${state.currentTemp}`;
+  changeTempColor(fahrenheit)
+}
+
+const state = {
+  currentTemp:  0,
+};
+
+// temperature
+const increaseTemp = () => {
+  state.currentTemp += 1;
+  const tempContainer = document.querySelector('#currentTemp');
+  tempContainer.textContent = `${state.currentTemp}`;
+};
+const decreaseTemp = () => {
+  state.currentTemp -= 1;
+  const tempContainer = document.querySelector('#currentTemp');
+  tempContainer.textContent = `${state.currentTemp}`;
+};
```

Comment:
There is shared code here and in getTempsByCity.  I recommend thinking about how you could refactor to DRY the code.